test/pycheck/prepared_test.py

```diff
@@ -125,8 +125,8 @@ def test_prepared_pipeline(conn: Connection):
         # the duckdb table that should fail because the insert into the heap
         # table opens an implicit transaction.
         with pytest.raises(
-            psycopg.errors.ActiveSqlTransaction,
-            match="DuckDB queries cannot be executed within a pipeline",
+            psycopg.errors.InternalError,
```

Comment:
It would be nice if we would do some errorcode conversions between DuckDB and Postgres errors. That seems like a task for a separate PR though.